src/main/java/seedu/address/model/interview/Interview.java

```diff
@@ -76,14 +76,15 @@ public boolean isDuringOfficeHour() {
         if (this.interviewDay > 5) {
             return false;
         }
-
-        if (interviewHour < 8) {
-            return false;
-        } else if (interviewHour >= 17 && interviewMin > 30) {
+        if (interviewHour < 8 || interviewHour > 17) {
```

Comment:
@lzan98 hmm just wondering, office hours we stipulated are 8am-6pm, i.e. the last interview can be scheduled at 17:30, will this cause a false return prematurely?